src/ui/shared/requests/useQuotes.ts

```diff
@@ -176,6 +177,7 @@ export function useQuotes({
   receiveTokenInput?: string;
   spendPosition: AddressPosition | EmptyAddressPosition | null;
   receivePosition: AddressPosition | EmptyAddressPosition | null;
+  sortType: QuoteSortType;
```

Comment:
Why is required here? It's not required in `QuotesParams`

Reply:
I think it should be required, just want things to be explicit

Reply:
updated